docs/guide/audit-trail.md

```diff
@@ -0,0 +1,71 @@
+
+### Alert Rule Changes
+
+Tracks when alerts are created, deleted, enabled, or disabled, as well as any updates made to [alert rules](https://sematext.com/docs/guide/alerts-guide/). This includes changes to alert conditions, queries, filters, schedules, names, priorities, group-by and roll-up settings, metrics, and transformation scripts. It also captures changes to notification settings and hooks.
+
+### Synthetics Monitoring
+
+Tracks creation, updates, disabling, and deletion of [monitors](https://sematext.com/docs/synthetics/). This includes:
+
+- Changes to monitor URLs, device types, run intervals, [locations](https://sematext.com/docs/synthetics/#locations), and conditions
+- [Status pages](https://sematext.com/docs/synthetics/status-pages/) and [scheduled pauses](https://sematext.com/docs/synthetics/scheduled-pauses/) being created, updated, or deleted
+- [SSL certificate tracking](https://sematext.com/docs/synthetics/ssl-certificate-monitoring/) and validation toggled on or off
+- Updates to request settings and authentication
+- Monitor type changes (URL/script-based), as well as script and secret variable updates
+
+### Pipelines and Processors
+
+Records creation, updates, removal, enabling, and disabling of [processors](https://sematext.com/docs/logs/processors-overview/). Tracks detailed changes to processor settings such as field mappings, filters, target/source fields, scripts, sampling rates, and processor ordering.
+
+###  Reports and Dashboards
+
+Tracks creation and updates of reports at both App and Dashboard levels, updates to [report components](https://sematext.com/docs/dashboards/reports-and-components/), and creation or deletion of [dashboards](https://sematext.com/docs/dashboards/).
+
+> When multiple settings are changed at once—for example, if several fields in an alert rule are edited—Audit Trail groups them into a single event. Each recorded entry shows what was changed, along with the previous and new values for each field.
+
+![Audit Trail Flyout](/docs/images/guide/audit-trail/audit-trail-event-flyout.png)
+
+## How to Use It
+
+You can access the Audit Trail from your Sematext Cloud account. It allows you to:
+
+- Filter events by user, event type, or related App
+- View the most frequent actions in the Top Events section
+- Click from an event to open the related App, Alert Rule, Pipeline, or other resource
+- See a timestamp and the user responsible for each action
+
+![Audit Trail Usage](/docs/images/guide/audit-trail/audit-trail-usage.gif)
+
+Audit Trail can be enabled or disabled at any time from your account settings.
```

Comment:
Only the user with Account OWNER role can do it? Should we say that?
Should we mention Team Accounts to encourage their use?